tasks/0-setup.yaml

```diff
@@ -264,6 +266,12 @@ spec:
         echo -n "${PERFORM_SCAN_TRIVY}">$(results.scan-trivy.path)
         echo -n "${PERFORM_SCAN_IBM}">$(results.scan-ibm.path)
 
+        if [[ -d workspace ]]; then
```

Comment:
Should check here is user specify input param and use it, if not then check for workspace

Reply:
Complete